src/main/java/uk/gov/homeoffice/digital/sas/timecard/configurations/KafkaTopicConfig.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.homeoffice.digital.sas.timecard.configurations;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+
+@Configuration
+public class KafkaTopicConfig {
+
+  @Bean
+  public NewTopic timecardTopicBuilder() {
+    return TopicBuilder.name("callisto-timecard")
```

Comment:
Might be worth pulling in the topic name from the application.properties. And the same for where we send the message.

Reply:
Moved the topic name into application.properties now

Reply:
Just on the naming of the new app property, it seems a bitt excessive. We could name it closer to something our application 'understands' e.g. `kafka-topic.timecard`
Feel free to just resolve if you disagree.

Reply:
Updated to `kafka.topic`